<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="displayObjects">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePoidsSac">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineValeur">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePoids">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPoids">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValeur">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAdd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add object</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPoidsSac">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sack capacity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDelete">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>511</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete selected object</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="displayTree">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Display tree (with cutting)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSolve">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Display best solution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAbout">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>590</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>590</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>511</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>How to use</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
